src/models/machine_learning/GlacierWideInv.jl

```diff
@@ -0,0 +1,57 @@
+export GlacierWideInv
+
+"""
+    GlacierWideInv{
+        ComponentVectorType <: ComponentVector
+    } <: PerGlacierModel
+
+Per glacier invertible parameter container.
+`GlacierWideInv` wraps a ComponentVector (θ) that stores one scalar parameter per glacier and implements the `PerGlacierModel` interface used by the inversion machinery.
```

Comment:
Should we be using this same util when doing the inversion with respect to the initial condition? That is a problem I encountered when implementing the IC inversion